packages/devops/scripts/ci/testInternalDependencies.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -le
```

Comment:
`-l` was not used here - just double checking that its addition is on purpose

By the way I'm not familiar with `-l`, what does it do?

Reply:
Jeez you've got good eyes! This was me testing that they'd work together, it's not harmful but also not required here, just tells bash to run the script in a regular user session (used during startup scripts called from crontab to make sure PATH, environment variables etc. are set up as normal)

Reply:
Removed :+1: